### PR TITLE
Update Copyright year to 2015 - Issue #489

### PR DESCRIPTION
**Pull request by Marczeeee**
Issue #489

### CODE REVIEW

**Comment by davvd** [comment]
Thanks, I'll ask someone to do a code review here, thanks for your pull request

**Comment by davvd** [comment]
@carlosmiranda this pull request is for you, please review

File: LICENSE.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
- Copyright (c) 2014, Thindeck.com
+ Copyright (c) 2015, Thindeck.com
```
Comment (carlosmiranda) [inline LICENSE.txt]: Let's use `2014-2015` instead.

**Comment by carlosmiranda** [comment]
@Marczeeee Sorry for the delay in responding. I have one comment above.

**Comment by Marczeeee** [comment]
@carlosmiranda Updated license dates to date interval.

**Comment by carlosmiranda** [comment]
@rultor let's merge please

**Comment by yegor256** [comment]
@rultor merge it

**Comment by davvd** [comment]
@carlosmiranda Thanks a lot, I just topped your account for **16 mins**, transaction ID `50597922`. added +16 to your rating, now it is equal to [+2043](http://www.netbout.com/b/35023?open=rating)

**Comment by davvd** [comment]
@rultor please deploy

**Pull request merged**